tests/testthat/test-job_list_utils.R

```diff
@@ -0,0 +1,53 @@
+box::use(
+  app/logic/job_list_utils[
+    process_job_data
+  ]
+)
```

Comment:
Please add a second `box::use` statement with `testthat` imports. It currently works without it, because `testthat` is a dependency of Rhino, but 1. this will increase robustness (so you will be able to trigger tests using other commands, not only `rhino::test_r`; 2. this will increase readability of the code.